src/Gui/MDIView.cpp

```diff
@@ -448,4 +448,14 @@ void MDIView::setCurrentViewMode(ViewMode mode)
     }
 }
 
+QString MDIView::buildWindowTitle()
+{
+    QString windowTitle = QString::fromLatin1("");
+    if (getGuiDocument()->isModified()) {
+        windowTitle = QString::fromLatin1("* ");
+    }
+
+    return windowTitle + QString::fromLatin1(getAppDocument()->getName());

```

Comment:
Currently, the internal name of a document is 7-bit ASCII but this will change in the future. So, you better use `QString::fromUtf8`

Reply:
@kadet1090 FYI